jaxws-retail/service/pom.xml

```diff
@@ -70,6 +77,7 @@
                 <artifactId>maven-war-plugin</artifactId>
                 <configuration>
                     <attachClasses>true</attachClasses>
+                    <packagingExcludes>WEB-INF/lib/cxf-*.jar</packagingExcludes>
```

Comment:
I guess this is related to not using the provided scope for the new dependency?

Reply:
Yes